### ISSUE TITLE
Official Helm Chart Offering

### ISSUE THREAD

**Issue by acelinkio** [rabbitmq/cluster-operator#1637]
**Is your feature request related to a problem? Please describe.**
The only helm chart offering for deploying cluster-operator is from Bitnami.

**Describe the solution you'd like**
Have an official helm chart offering, outside of Bitnami.

**Describe alternatives you've considered**
GitOps the cluster-operator.yaml mentioned in https://www.rabbitmq.com/kubernetes/operator/install-operator#installation or not use the cluster-operator at all.

**Additional context**
In the last several months, Broadcomm purchased Bitnami's parent company VMWare.  Broadcomm has been making substantial changes to their business models.  Would appreciate having an official chart to reduce the reliance upon 3rd parties.

**Comment by NikoGrano** [comment]
@MirahImage Sorry to bounce into an old issue. What about now? 
The [Bitnami chart](https://github.com/bitnami/charts/blob/main/bitnami/rabbitmq-cluster-operator/Chart.yaml) is dependent on images, which will not be available after changes. Will there be a chart available for this operator without any third-party dependencies? 

@acelinkio I don't understand how you saw this coming before anyone else... Good job.

**Comment by ednxzu** [comment]
I can only second what everyone else here already raised, the situation change with bitnami makes it hard to justify not providing an official helm chart at this point.

**Comment by lukebakken** [comment]
I'm sure Team RabbitMQ would be thrilled if people who _need_ this step up and provide it.

**Comment by NikoGrano** [comment]
@lukebakken
> I'm sure Team RabbitMQ would be thrilled if people who _need_ this step up and provide it.

Yes, but before moving forward with any implementation, **it would be helpful to get a green light to ensure the effort isn’t wasted.** As @MirahImage mentioned earlier, the official Helm chart currently appears to be the Bitnami version. Since RabbitMQ is owned by Broadcom, it's possible that there are internal discussions or future plans already in motion.

To avoid duplicating work or heading in a direction that may not align with the broader roadmap, it would be great to get some clarification on the current status and direction.

@MirahImage, would you be able to shed some light on this? Specifically:
- Will the Bitnami chart remain the official one?
- If not, should it be copied into this repository with required changes?
- Or is the intention to develop a new chart from scratch?

Clarifying this would really help a lot...
